test/unit/alignment/pairwise/edit_distance/semi_global_edit_distance_max_errors_unbanded_test.cpp

```diff
@@ -30,11 +25,77 @@ using semi_global_edit_distance_max_errors_unbanded_types
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s3u_15u_e0, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s3u_15u_e0, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s3u_15u_e0, uint64_t>,
+
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e255, uint8_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e255, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e255, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e255, uint64_t>,
+
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e0, uint8_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e0, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e0, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_02_s1u_15u_e0, uint64_t>,
+
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e255, uint8_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e255, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e255, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e255, uint64_t>,
+
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e5, uint8_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e5, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e5, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_01T_s17u_1u_e5, uint64_t>,
+
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e255, uint8_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e255, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e255, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e255, uint64_t>,
+
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e0, uint8_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e0, uint16_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e0, uint32_t>,
+        semi_global_fixture<&semi_global::edit_distance::max_errors::unbanded::dna4_03_e0, uint64_t>
```

Comment:
and this compiles in reasonable time?

Reply:
Would it make sense to split this into multiple translation units if not?

Reply:
later :)